### PR TITLE
Arrange code & set the code conventsion #1

### PR DESCRIPTION
**Pull request by AmosDoan**
**Issue** #1 
**Summary**
- Apply Lombok for the basic class
- Change tab to 4 spaces

### CODE REVIEW

**Comment by AmosDoan** [comment]
@YoseobKim I'm not test yet. I will finish test and notify to you. Until that time, please just review not merge (_ _)

File: src/main/java/com/tokyoprogrammer/aqitracer/configure/BatchConfiguration.java [hunk 1]
```diff
@@ -25,88 +25,91 @@
...
+    @Value("batch.baseUrl")
+    private static String BASE_URL;
+
+    @Value("batch.tokenUrl")
+    private static String TOKEN_URL;
+
+    // The IDS_LIST comes from the result of the URL below.
+    // TODO The list should be dynamically generated using the API below.
+    // http://api.waqi.info/search/?token=80860687b4b6c03df2a8c5e1b9a895275abc368d&keyword=Seoul
+    private static final String IDS_LIST[] = { "5508", "4487", "1690", "1673", "1679", "4498" };
```
Comment (YoseobKim) [inline src/main/java/com/tokyoprogrammer/aqitracer/configure/BatchConfiguration.java:55]: I think this list should be generated automatically using the API.
FYI. https://aqicn.org/json-api/doc/
Search API would be proper for this purpose.

File: build.gradle [hunk 2]
```diff
@@ -32,4 +32,5 @@ dependencies {
 	runtimeOnly('com.h2database:h2')
 	testImplementation('org.springframework.boot:spring-boot-starter-test')
 	testImplementation('org.springframework.batch:spring-batch-test')
+	implementation('org.projectlombok:lombok')
```
Comment (YoseobKim) [inline build.gradle:35]: Good!

File: src/main/java/com/tokyoprogrammer/aqitracer/listener/JobCompletionNotificationListener.java [hunk 3]
```diff
@@ -1,30 +1,29 @@
...
-		if(jobExecution.getStatus() == BatchStatus.COMPLETED) {
-			log.info("!!! JOB FINISHED! Time to verify the results");
-			// TODO Add verification logic here.
-		}
-	}
+    @Override
+    public void afterJob(JobExecution jobExecution) {
+        if(jobExecution.getStatus() == BatchStatus.COMPLETED) {
+            log.info("!!! JOB FINISHED! Time to verify the results");
+            // TODO Add verification logic here.
```
Comment (YoseobKim) [inline src/main/java/com/tokyoprogrammer/aqitracer/listener/JobCompletionNotificationListener.java:26]: Later on, we'd better add verification logic in here whether the job has been finished successfully or not.

File: src/main/java/com/tokyoprogrammer/aqitracer/reader/RestAQIReader.java [hunk 4]
```diff
@@ -23,15 +26,19 @@ public RestAQIReader(List<String> apiUrl, RestTemplate restTemplate) {
...
 	// Read every API Result from multiple urls
 	private void prepareResponsesList() {
 		data = new ArrayList<>();
 		for(String url : apiUrl) {
 			ResponseEntity<RestResultJSON> response = restTemplate.getForEntity(url, RestResultJSON.class);
-			data.add(response.getBody().getData());
+			try {
+				data.add(response.getBody().getData());
+			} catch (NullPointerException e) {
+			    log.warn("Response has empty body data");
```
Comment (YoseobKim) [inline src/main/java/com/tokyoprogrammer/aqitracer/reader/RestAQIReader.java:40]: I think this line also should be aligned with others. 4 spaces.

**Comment by AmosDoan** [comment]
I tested with the changes of this PR and checked like below. (Get the information from API and converting to model class.
@YoseobKim , could you review this PR? After you reviewed and notify to me okay to merge, then let me squash the commits
```
2018-12-15 16:24:22.460  INFO 92701 --- [           main] c.t.a.processor.PersonItemProcessor      : Converting (AQIndexJSON [idx=5508, aqi=139, city=AQIndexStationJSON [name=Seoul (서울), url=https://aqicn.org/city/seoul, geo=[37.566535, 126.9779692]], time=AQIndexTimeJSON [stime=2018-12-15 16:00:00, tz=+09:00, vtime=1544889600]]) into (AQIndexDB [uid=5508, aqi=139, cityName=Seoul (서울), vtime=1544889600])
2018-12-15 16:24:22.460  INFO 92701 --- [           main] c.t.a.processor.PersonItemProcessor      : Converting (AQIndexJSON [idx=4487, aqi=107, city=AQIndexStationJSON [name=Jongro, Seoul, South Korea (대한민국종로 서울), url=https://aqicn.org/city/korea/seoul/jongro, geo=[37.570633, 126.996783]], time=AQIndexTimeJSON [stime=2018-12-15 16:00:00, tz=+09:00, vtime=1544889600]]) into (AQIndexDB [uid=4487, aqi=107, cityName=Jongro, Seoul, South Korea (대한민국종로 서울), vtime=1544889600])
2018-12-15 16:24:22.460  INFO 92701 --- [           main] c.t.a.processor.PersonItemProcessor      : Converting (AQIndexJSON [idx=1690, aqi=127, city=AQIndexStationJSON [name=Jung-gu, Seoul, South Korea (대한민국중구 서울), url=https://aqicn.org/city/korea/seoul/jung-gu, geo=[37.564639, 126.975961]], time=AQIndexTimeJSON [stime=2018-12-15 16:00:00, tz=+09:00, vtime=1544889600]]) into (AQIndexDB [uid=1690, aqi=127, cityName=Jung-gu, Seoul, South Korea (대한민국중구 서울), vtime=1544889600])
2018-12-15 16:24:22.460  INFO 92701 --- [           main] c.t.a.processor.PersonItemProcessor      : Converting (AQIndexJSON [idx=1673, aqi=102, city=AQIndexStationJSON [name=Guro-gu, Seoul, South Korea (대한민국구로구 서울), url=https://aqicn.org/city/korea/seoul/guro-gu, geo=[37.480178, 126.892606]], time=AQIndexTimeJSON [stime=2018-12-15 16:00:00, tz=+09:00, vtime=1544889600]]) into (AQIndexDB [uid=1673, aqi=102, cityName=Guro-gu, Seoul, South Korea (대한민국구로구 서울), vtime=1544889600])
2018-12-15 16:24:22.460  INFO 92701 --- [           main] c.t.a.processor.PersonItemProcessor      : Converting (AQIndexJSON [idx=1679, aqi=97, city=AQIndexStationJSON [name=Mapo-gu, Seoul, South Korea (대한민국마포구 서울), url=https://aqicn.org/city/korea/seoul/mapo-gu, geo=[37.550133, 126.945167]], time=AQIndexTimeJSON [stime=2018-12-15 16:00:00, tz=+09:00, vtime=1544889600]]) into (AQIndexDB [uid=1679, aqi=97, cityName=Mapo-gu, Seoul, South Korea (대한민국마포구 서울), vtime=1544889600])
2018-12-15 16:24:22.461  INFO 92701 --- [           main] c.t.a.processor.PersonItemProcessor      : Converting (AQIndexJSON [idx=4498, aqi=80, city=AQIndexStationJSON [name=Gil-dong, Seoul, South Korea (대한민국길동 서울), url=https://aqicn.org/city/korea/seoul/gil-dong, geo=[37.534035, 127.139172]], time=AQIndexTimeJSON [stime=2018-12-15 16:00:00, tz=+09:00, vtime=1544889600]]) into (AQIndexDB [uid=4498, aqi=80, cityName=Gil-dong, Seoul, South Korea (대한민국길동 서울), vtime=1544889600])
```

**Review by YoseobKim — APPROVED** [review]
Thank you.

**Pull request merged**